queuebot/cogs/queue/__init__.py

```diff
@@ -315,7 +327,7 @@ def wait_check(msg):
 
             vs_message = await queue.send(emote_sequence)
             for this_emoji in temp_emotes:
-                await vs_message.add_reaction(this_emoji)
+                await vs_message.add_reaction(NUMERIC_EMOTES[temp_emotes.index(this_emoji)])
```

Comment:
It would be better to switch this into using `enumerate` rather than doing index lookups for each emoji.